src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinRTWebSocket.cs

```diff
@@ -105,6 +114,20 @@ public override string SubProtocol
                 websocketControl.SupportedProtocols.Add(subProtocol);
             }
 
+            if (MessageWebSocketClientCertificateSupported)
```

Comment:
Is your suggestion to throw an exception if _MessageWebSocketClientCertificateSupported_ is **false**? If so, that sounds like a breaking change (ClientWebSocket.ConnectAsync would stop working outside of Insider Preview builds).